app/src/main/java/org/simple/clinic/illustration/HomescreenIllustrationRepository.kt

```diff
@@ -13,13 +13,13 @@ import java.io.File
 import javax.inject.Inject
 
 class HomescreenIllustrationRepository @Inject constructor(
-    private val illustrationDao: HomescreenIllustration.RoomDao,
+    private val illustrations: List<HomescreenIllustration>,
```

Comment:
As mentioned above, doing this will make the illustrations be read and parsed at the time of injection, which is on the UI thread. Can we inject the `ConfigReader` here instead and load it at the time of invoking the method?